### ISSUE TITLE
Embedding Model does not support custom baseurl

### ISSUE THREAD

**Issue by hhtower** [ItzCrazyKns/Perplexica#399]
![image](https://github.com/user-attachments/assets/b02dd63b-a5ff-44a1-a269-17149f66294e)
Can add a custom_openai?

**Comment by babytdream** [comment]
same question. Do you fix it? thanks @hhtower

**Comment by bpawnzZ** [comment]
I agree. Embeddings should allow custom openai compatible endpoints. Would be nice if I could direct embeddings to LiteLLM like the model

**Comment by ChaojuWang** [comment]
folks any update?
